src/common/datasource/Cargo.toml

```diff
@@ -24,7 +24,7 @@ datafusion.workspace = true
 derive_builder = "0.12"
 futures.workspace = true
 object-store = { path = "../../object-store" }
-orc-rust = { git = "https://github.com/WenyXu/orc-rs.git", rev = "0319acd32456e403c20f135cc012441a76852605" }
+orc-rust = "0.2.4"
```

Comment:
Does `orc-rust = "0.2` automatically choose the latest version for you?

Reply:
Updated